packages/notification-center/src/components/notification-center/components/notification-item/NotificationListItem.tsx

```diff
@@ -206,6 +206,7 @@ export function NotificationListItem({
           shadow={colorScheme === 'dark' ? shadows.dark : shadows.light}
           withArrow
           classNames={overrideClasses}
+          withinPortal={true}
```

Comment:
```suggestion
          withinPortal={true}
          middlewares={{ flip: false, shift: false }}
```
Add this line too. This is needed to handle a bug in Mantine where in some cases it crashes the whole page.